tests/cli-preprocessor.test.ts

```diff
@@ -25,3 +25,10 @@ test('knip --preprocessor knip-preprocessor', () => {
 test('knip --preprocessor @org/preprocessor', () => {
   assert.equal(exec('knip --preprocessor @org/preprocessor'), 'hi from scoped preprocessor');
 });
+
+test(`knip --preprocessor with-args --preprocessor-options {"food":"cupcake"}`, () => {
+  assert.equal(
+    exec(`knip --preprocessor with-args --preprocessor-options {\\"food\\":\\"cupcake\\"}`),
```

Comment:
Perhaps more clear to use single quotes over escaping the double quotes? I don't mind it in general for tests, but some people use them as an example.

Reply:
Not sure if this is due to a Windows shell oddity, but this was the only way I was able to get the tests actually working

Reply:
Alright no problem